src/braket/aws/aws_quantum_task.py

```diff
@@ -270,7 +275,7 @@ def result(self) -> Union[GateModelQuantumTaskResult, AnnealingQuantumTaskResult
             if the task completed successfully; returns None if the task did not complete
             successfully or the future timed out.
         """
-        if self._result or self._status(True) in self.NO_RESULT_TERMINAL_STATES:
+        if self._result or self._status in self.NO_RESULT_TERMINAL_STATES:
```

Comment:
What's up with this change? I couldn't find self._status anywhere.

Reply:
My bad, I tried introducing a `_status` property and removed it